### ISSUE TITLE
Duplicate resource cards in podcardlist. 

### ISSUE THREAD

**Issue by PiotrDabkowski** [kubernetes/dashboard#1161]
In `src/app/frontend/podlist/podcardlist.html` there are 2 versions of kd-resource-card defined:

```html
<kd-resource-card ng-if="::$ctrl.podListResource"  ... >
...
<kd-resource-card ng-if="::!$ctrl.podListResource" ... >
...
```
It would be nice to unify the way we display podlists.

**Comment by floreks** [comment]
This is done to use frontend pagination if backend pagination can not be used. Same pattern is applied to every resource card list. Pods however don't have `podcard` component.

**Comment by bryk** [comment]
@floreks Is it possible that, at some point, we remove the other version of the table and just have pagination-everywhere?

Or should we just refactor pods into podcard component?

**Comment by floreks** [comment]
We can enforce backend pagination and make `resourceList` parameter required. We could then skip frontend pagination at all and rely on backend only. Currently events are still using frontend pagination. I think it is last place that needs adjustment. After that we can remove frontend pagination support if needed.

**Comment by PiotrDabkowski** [comment]
Solved by #1192
